js/bundle.js

```diff
@@ -212,14 +217,14 @@ function clickGo() {
         overlays.clearLayers();
 
         if (type == 'node') {
-            for (i = 0; i < object.length; i++) {
+            for (i = object.length -1; i >= 0; i--) {
                 if (object[i].visible) {
-                    overlays.addLayer(L.circleMarker([object[i].lat, object[i].lon]));
+                    overlays.addLayer(L.marker([object[i].lat, object[i].lon]).bindTooltip((i+1).toString(), {permanent: true}));
```

Comment:
In the sample you gave, the tooltips stack on top of one another and make it hard to figure out which node versions are where. I wonder if instead of showing all the tooltips to start with, we only showed the tooltips when the user clicks on an icon in the version's column.

Reply:
Indeed at first I made it in the way you advise: show version tooltip only when mouse hover. However I found it's difficult find out how a node is moved between versions. Especially when there are a lot of moves, have to hover one by one to see where a specific version of node is.
Maybe a better approach is to show tooltip of a version when mouse click/hover on that version cell in table. I'll try to do it later. I'm now working on the visualisation of way history on map.

Reply:
Yea, it might make more sense to remove the map stuff altogether until the node and way history work better. I had hopes of supporting nodes and ways at the beginning but ended up  leaving the node history map in a half-finished state :(.